src/libs/EmojiUtils.js

```diff
@@ -70,6 +70,30 @@ function isSingleEmoji(message) {
     return matchedUnicode === currentMessageUnicode;
 }
 
+/**
+ * Validates that this message contains only emojis
+ *
+ * @param {String} message
+ * @returns {Boolean}
+ */
+function containsOnlyEmojis(message) {
+    const match = message.match(CONST.REGEX.EMOJIS);
+    if (match) {
+        const codes = [];
+
```

Comment:
Could you please match the isSingleEmoji function by returning early? 

Also, I wonder if the two functions could be combined in any way?

Reply:
> Could you please match the isSingleEmoji function by returning early?

Done 
> Also, I wonder if the two functions could be combined in any way?

You mean `isSingleEmoji` and `containsOnlyEmojis`?
The only part the share is `const match = message.match(CONST.REGEX.EMOJIS)`, we could move it into a separate method

Reply:
Thanks. Okay, forget the second part then.